### ISSUE TITLE
Color chooser's eyedroppers not working

### ISSUE THREAD

**Issue by Odyseus** [linuxmint/Cinnamon#5633]
Hello, everybody.

The eyedroppers of color choosers inside applet setting windows aren't working. They don't actually choose a color. The only responsive key is **Escape**, to exit color choosing mode. Other eyedroppers color pickers seems to work fine (only tested **Gimp** color picker).

This happens on an installed system and on the live system, when booted from the ISO on a virtual machine (not tested on a physical machine).

#### How to reproduce

- Place the Cinnamon applet called **Settings API Demo** (it comes by default on Cinnamon 3.0.7) on the panel.
- Go to its settings window and click the eyedropper.
- None of the mouse clicks are responsive (color on screen is not selected).
- Click **Escape** to exit color choosing mode.

### My system information
- **Operating System:** Linux Mint 18 Cinnamon 64-bit
- **Cinnamon Version:** 3.0.7
- **Linux Kernel:** 4.4.0-21-generic
- **Processor:** AMD Phenom(tm) II X4 955 Processor × 4
- **Memory:** 6.8 GiB
- **Hard Drives:** 1936.6 GB
- **Graphics Card:** NVIDIA Corporation G94 [GeForce 9600 GT]

**Comment by JosephMcc** [comment]
Thanks for reporting but this isn't really relevant anymore in git master. The settings API has seen a complete revamp for Cinnamon 3.2

**Comment by Odyseus** [comment]
Hello, @JosephMcc.

Thanks for the information. I wanted to wait to answer to this issue until I had the oportunity test the Cinnamon version from this repository.

So I installed Cinnamon from this repository into one of my VirtualBox machines with Linux Mint 18 Cinnamon installed. You were right. The revamp on the settings system (which is totally awesome, by the way) made this issue totally irrelevant, because the eyedropper was removed from the color choosers.

I found several issues with the new settings system, but first, I will do more tests and research the current issues before reporting them.

Thanks, JosephMcc.
